### PR TITLE
FIX improving performance tuning documentation based on the recent modifications to MHD usage

### PR DESCRIPTION
**Pull request by fgalan**
Documentation items regarding #2724 :

* The section regarding thread model has been re-written. The part related with file descriptors has been moved to its own section.
* Some coments related with poll() and epoll() in the section about MHD tunning
* (Pending) Comments regarding the relationship between -maxConnections and -reqPoolSize (a question has been done in the MDH maliling list). Probably the figure currently there is incorrect.

Documentation items regarding #2166:

* Comment regarding CLOSE_WAIT are ok, as they are part of libcurl cache strategy

### CODE REVIEW

File: doc/manuals/admin/perf_tuning.md [hunk 1]
```diff
@@ -130,32 +135,65 @@ high load scenarios. The other two parameters usually work well with their defau
...
+  in this pool. See [HTTP server tuning section](#http-server-tuning) in this page for more information.
+* Notifications pool. Set by `-notificationMode threadpool:q:n`, being `n` the number of threads in this pool.
+  See [notification modes and performance section](#notification-modes-and-performance) in this page.
+
+Using both parameters, in any situation (either idle or busy) Orion consumes a fixed number of threads:
+
+* Main thread (the one that starts the broker, then sleeps forever)
+* Subscription cache synchronization thread (if `-noCache` is used then this thread is not created)
+* `c` listening thread for the IPv4 server (if `-ipv6` is used then these threads are not created)
+* `c` listening threaftestd for the IPv6 server (if `-ipv4` is used then these threads are not created)
```
Comment (crbrox) [inline doc/manuals/admin/perf_tuning.md]: typo? `threaftestd` -> `threads`

Comment (fgalan) [inline doc/manuals/admin/perf_tuning.md]: Fixed in 8b566fbe4f49d0584cb403b472dad5dba6e8afb2

**Comment by crbrox** [comment]
LGTM

**Comment by kzangeli** [comment]
LGTM

**Comment by fgalan** [comment]
PR was merged while this is still pending

> (Pending) Comments regarding the relationship between -maxConnections and -reqPoolSize (a question has been done in the MDH maliling list). Probably the figure currently there is incorrect.

It will be done in a next PR.

**Pull request merged**